Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -66,6 +72,17 @@ class NotificationsViewController: UIViewController {
             make.bottom.equalTo(view.safeAreaLayoutGuide)
         }
     }
+
+    private func setupEmptyStateView() {
+        emptyStateView.setup(image: UIImage.notifIcon!,
+                             title: "No new notifications",
+                             subtitle: "You’re all caught up at the moment!")
```

Comment:
nit but this is a multiline argument violation in terms of linting. usually the parentheses should be in a separate line.